src/pygeosx/pygeosx.cpp

```diff
@@ -353,6 +364,20 @@ PyInit_pygeosx()
     return nullptr;
   }
 
+  LvArray::python::PyObjectRef<> pysolverModule = geosx::python::PyInit_pysolver();
+  Py_XINCREF( pysolverModule );
```

Comment:
I think it would be clearer here if you got rid of the `Py_XINCREF` and replaced it with `pysolverModule.release()` in the next line, since we're transferring ownership of the module to the interpreter.

Reply:
Changed it to be `Types` in the `pygeosx` module